docs/examples/writing_drivers/Creating-Instrument-Drivers.ipynb

```diff
@@ -307,7 +524,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## DLL-based instruments\n",
+    "## DLL-based instruments (Conventional style)\n",
```

Comment:
i'd suggest to not mark these 3 sections (VisaInstrument and DLL based isnrtuments) as "conventional style" because these sections are not so much about ``self.add_parameter`` but more about how an instrument driver looks like in general
```suggestion
    "## DLL-based instruments\n",
```